pages/_app/index.jsx

```diff
@@ -143,11 +142,12 @@ class App extends NextApp {
     const { store } = this
     const { Component, pageProps } = this.props
     const { isAuthorized } = this.state
+    const pathname = this.props.router.asPath
```

Comment:
Does not looks good, to be honest.

Reply:
you can still destructure it in two lines above.

Reply:
It's not about desctructuring. I did not want this for a reason, the purpose of this line is slightly different from the above ones.

I do not like passing pathname to application. It should be solved in another way but I do not know how exactly it should be.